Resources/Prototypes/_Shitmed/Entities/Structures/Walls/walls.yml

```
@@ -31,8 +31,8 @@
     damageModifierSet: StructuralMetallic
   - type: Physics
     bodyType: Static
-  - type: Reflect
-    reflectProb: 1
+  #- type: Reflect # Mono - remove reflect
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)